### ISSUE TITLE
sit devices are allowed to have no local IP

### ISSUE THREAD

**Issue by Hello71** [systemd/systemd#12041]
**systemd version the issue has been seen with**
regression with #10717

**Used distribution**
Gentoo Linux

**Expected behaviour you didn't see**
device is created

**Unexpected behaviour you saw**
```
he-ipv6: vti/ipip/sit/gre/gretap/erspan tunnel without a local IPv4 address configured in /etc/systemd/network/he-ipv6.netdev. Ignoring
```

**Steps to reproduce the problem**
```
[NetDev]
Name=he-ipv6
Kind=sit

[Tunnel]
Remote=[ip]
TTL=255
```

**Additional information**
it works with `ip tunnel add he-ipv6 mode sit remote [ip]` and results in `he-ipv6: ipv6/ip remote [ip] local any ttl inherit`

**Comment by Hello71** [comment]
@yuwata

**Comment by yuwata** [comment]
Indeed. I will fix this soon. Thank you for the report.

**Comment by yuwata** [comment]
Fix is waiting in #12066. If possible, please test the PR.
